packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_ETH.sol

```diff
@@ -76,4 +80,33 @@ contract OVM_ETH is L2StandardERC20, IWETH9 {
         // Other than emitting an event, OVM_ETH already is native ETH, so we don't need to do anything else.
         emit Withdrawal(msg.sender, _wad);
     }
+
+
+    /**********************************
+     * Overridden ERC20 Functionality *
+     **********************************/
+
+    /**
+     * This override allows for OVM_ETH to be sent to address(0), which is not normally permitted by the
+     * OZ ERC20 implementation, but IS possible for EVM calls to the zero address with nonzero value.
+     * @param _sender Address sending the OVM_ETH.
+     * @param _recipient Address recieving the OVM_ETH.
+     * @param _amount The amount of OVM_ETH being sent.
+     */
+    function _transfer(
+        address _sender,
+        address _recipient,
+        uint256 _amount
+    )
+        internal
+        virtual
+        override
+    {
+        // super._transfer disallows this condition, but we can just interpret it as a burn.
+        if (_recipient == address(0)) {
+            _burn(_sender, _amount);
```

Comment:
The PR description says: 
> Small little PR to improve compatibility with the EVM.

If the goal is to mimic the EVM, then IMO this should be `_transfer(_sender, address(0), _amount)`. ie. the ETH isn't destroyed, it's just sent to `address(0)` where it's irretrievable. 

For the same reason, I think that the term burn is misleading where it occurs in elsewhere in this PR.

Reply:
```suggestion
        uint256 senderBalance = _balances[sender];
        require(senderBalance >= amount, "ERC20: transfer amount exceeds balance");
        unchecked {
            _balances[sender] = senderBalance - amount;
        }
        _balances[recipient] += amount;

        emit Transfer(sender, recipient, amount);
```